### ISSUE TITLE
AUDIO and IMG tags unusable if click ok on smart input dialog without choosing file

### ISSUE THREAD

**Issue by aerwin** [maqetta/maqetta#2002]
1. Drag AUDIO or IMG tag onto canvas
2. Click ok on smart input without choosing a src file
3. Note AUDIO or IMG element shrinks to size 0 and double-clicking won't bring up smart input

The AUDIO or IMG element can be selected in the outline view, but no way to get smart input to come up.

This does not happen with EMBED or VIDEO tags because they maintain a size even if no src is specified.

**Comment by peller** [comment]
didn't we use min-width/height CSS for some other widgets to work around this problem?

**Comment by aerwin** [comment]
@JonFerraiolo @jhpedemonte 

Raising priority of this one based on Jon's opening of duplicate #2150 as high priority.

**Comment by aerwin** [comment]
It looks like this even happens now if you select a valid file, so it's become more pervasive. The "controls" attribute no longer stays selected.

**Comment by aerwin** [comment]
Must fix M6

**Comment by aerwin** [comment]
AUDIO widgets will only have a size if the `controls` attribute is set to "controls" (meaning play, stop, etc. buttons should be shown). Despite this being specified in package.json, `controls` is always set to the empty string at runtime. This is different than the original scenario described in this ticket (where sizing issue isolated to when no `src` attribute was specified at all).

My early suspicion was that there was something specific to `controls` and its default value that was broken. But, as I investigated deeper, I saw that `controls` is defaulted correctly at runtime, but as soon as `src` is specified `controls` is wiped out. 

Slowly taking it further, I noticed that using the properties palette to set any other attribute of AUDIO (`loop`, `autoplay`, etc.) caused other settings to be wiped out. Then, as I tried other HTML widgets, I see this happens across the board.

So, this is obviously a much worse and widespread issue that's really separate from the core problem of the original ticket. Therefore, I'm going to spin-off a separate issue because even once that's resolved, the orignal (less severe and pervasive) sizing problem when no src is specified for AUDIO will still need to be addressed.
